tests/range/jsfmt.spec.js

```diff
@@ -1 +1,18 @@
-run_spec(__dirname, { rangeStart: 180, rangeEnd: 220 });
+run_file(__dirname, "range.js", { rangeStart: 224, rangeEnd: 227 });
+run_file(__dirname, "try-catch.js", { rangeStart: 7, rangeEnd: 8 });
+run_file(__dirname, "ignore-indentation.js", {
+  rangeStart: 222,
+  rangeEnd: 225
```

Comment:
This is super brittle and hard to manage, since you are adding a new run_file abstraction, I would recommend embedding some range logic:

Add a special token for the begin and end of the range like `<<(cursor)` `>>(cursor)` and turn them into numeric ranges and remove them before passing them to the test.